girder_annotation/docs/annotations.rst

```diff
@@ -185,6 +185,9 @@ near by values aggregate together when viewed.
     "normalizeRange": true             # If true, the rangeValues are normalized to [0, 1].  If
                                        # false, the rangeValues are in the
                                        # value domain.  Defaults to true.  Optional
+    "scaleWithZoom": true              # If true, scale the size of points with the zoom level of
```

Comment:
Can you add a comma after `"normalizeRange": true` for consistency with the rest of the examples?